omnisci_olio/workflow/client.py

```diff
@@ -23,11 +23,16 @@
     # pre-2.0 ibis and ibis_omniscidb
     from ibis_omniscidb import OmniSciDBClient as OmniSciDBBackend
 
+# Prefect's v1 logging used to use a Python logger when not in a Flow/Task context. In v2 it throws an exception.
+try:
```

Comment:
can we put this try, except block inside logger function? why is because, if we put it outside, it always uses the python logging even if there any flow exists.